src/erlcloud_ec2.erl

```
@@ -2415,6 +2420,178 @@ request_spot_instances(Request, Config) ->
+
+-spec(describe_spot_fleet_instances/3 :: (string(), string(), pos_integer()) -> {ok, [proplist()]} | {error, term()}).
+describe_spot_fleet_instances(SpotFleetRequestId, NextToken, MaxResults) ->
+    describe_spot_fleet_instances(SpotFleetRequestId, NextToken, MaxResults, default_config()).
+
+-spec(describe_spot_fleet_instances/4 :: (string(), string(), pos_integer(), aws_config()) -> {ok, [proplist()]} | {error, term()}).
+describe_spot_fleet_instances(SpotFleetRequestId, NextToken, MaxResults, Config) ->
+    Params = [
+        {"SpotFleetRequestId", SpotFleetRequestId},
+        {"NextToken", NextToken},
+        {"MaxResults", MaxResults}
+    ],
+    case ec2_query2(Config, "DescribeSpotFleetInstances", Params, ?NEW_API_VERSION) of
+        {ok, Doc} ->
+            {ok, [extract_describe_spot_fleet_request(Item) ||
+                Item <- xmerl_xpath:string("/DescribeSpotFleetInstancesResponse/activeInstanceSet/item", Doc)]};
+        {error, _} = E -> E
+    end.
+
+-spec(modify_spot_fleet_request/3 :: (string(), non_neg_integer(), default | no_termination) -> ok | {error, term()}).
+modify_spot_fleet_request(SpotFleetRequestId, TargetCapacity, ExcessCapacityTerminationPolicy) ->
+    modify_spot_fleet_request(SpotFleetRequestId, TargetCapacity,
+        ExcessCapacityTerminationPolicy, default_config()).
+
+-spec(modify_spot_fleet_request/4 :: (string(), non_neg_integer(), default | no_termination, aws_config()) -> ok | {error, term()}).
+modify_spot_fleet_request(SpotFleetRequestId, TargetCapacity, ExcessCapacityTerminationPolicy, Config) ->
+    Params = [
+        {"ExcessCapacityTerminationPolicy", encode_termination_policy(ExcessCapacityTerminationPolicy)},
+        {"SpotFleetRequestId", SpotFleetRequestId},
+        {"TargetCapacity", TargetCapacity}],
+
+    case ec2_query2(Config, "ModifySpotFleetRequest", Params, ?NEW_API_VERSION) of
+        {ok, _Doc} -> ok;
```

http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_ModifySpotFleetRequest.html this request may fail so please return the actual error in the doc